documentation/frontend/reference/testing_guidelines.md

```diff
@@ -357,15 +357,43 @@ output of failed tests by finding the `trace.zip` for the relevant test in the
 [Playwright Trace Viewer](https://playwright.dev/docs/trace-viewer) to inspect
 these (or open the zip yourself and poke around the files on your own).
 
-For Playwright tests failing in CI, a GitHub comment will appear with a link to
-download an artifact of the `test-results` folder.
+Playwright tests in CI are run with `-u` option by default, this means that
+snapshots will automatically be updated for modified parts of the UI if
+Playwright detects that. See
+[Updating snapshots](/frontend/guides/test.md#updating-snapshots) for more
+reading about this.
+
+When this happens, a GitHub comment will appear with a link to download zipped
+artifacts named in the form `*_snapshot_diff.zip`. Once downloaded, they can be
+decompressed and applied to your working branch by running:
+
+`ov unzip -p *_snapshot_diff.zip | git apply`
+
+The above command basically uses the `unzip` tool to unpack the contents of the
+downloaded archive file named `*_snapshot_diff.zip`, the `-p` option prevents it
+from actually creating any extracted file but rather prints the contents to the
+standard output which is then piped to `git apply`.
```

Comment:
Just a nit, and something we need to address more widely in our documentation (it's unfortunately pervasive, including in my own writing), but this would read a lot more clearly in active voice, rather than passive, and even more so as imperative instructions. "Once downloaded, apply them to your working branch by running", etc.